src/sentry/runner/commands/devserver.py

```diff
@@ -52,6 +52,18 @@
         "--synchronize-commit-group=transactions_group",
         "--no-strict-offset-reset",
     ],
+    "post-process-forwarder-generic": [
```

Comment:
We also have `generic-metrics` so I think it'll be good to call this something more specific for example:
`post-process-forwarder-generic-events`, `post-process-forwarder-issue-platform`, `post-process-forwarder-search-issues`
(unrelated but i've seen variants of these names floating around and i'm not sure which is the canonical name - can things be standardised on one?)